docker/docker-compose.yml

```diff
@@ -4,15 +4,23 @@ services:
   # Mainflux services
   users:
     image: mainflux/users:0.10.0
+<<<<<<< HEAD
```

Comment:
These conflict tags shouldn't be in the submitted code.

Reply:
I removed them in the commit [10dcbfd](https://github.com/CESARBR/knot-cloud-storage/commit/10dcbfd55691901be7c024fc35ac3fcf4419b2ec).

Reply:
Good @asf89 ! But these fixes should be in this commit 😉 Let us know if you have some git questions 😃

Reply:
@esdrasjnr you mean I should put the fix in the commit [862fe11](https://github.com/CESARBR/knot-cloud-storage/pull/41/commits/862fe114c39394e03d5cb166e394b0b2fab027b9)?

Reply:
@esdrasjnr Done. I spoke with @netoax and it was considered OK the fix in the actual commit.